File: Week 1/mario-more/mario.c
// Jorge Daniel Gómez Quintana "Mario"
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height, i, j;
    // Prompt height
    do
    {
        printf("What is the height of the pyramid ");
        height = get_int("What is the height of the pyramid ");

        if (height <= 0)
        {
            printf("Repeat: Height must be a positive number\n");
        }
    }
    while (height <= 0);

    for (i = 1; i <= height; i++)
    {
        for (j = 0; j < height - i; j++)
        {
            printf(" ");
        }
        // Left pyramid
        for (j = 0; j < i; j++)
        {
            printf("#");
        }
        // Gap between
        printf("  ");
        // Right pyramid
        for (j = 0; j < i; j++)
        {
            printf("#");
        }
        printf("\n");
    }

    return 0;
}

File: Week 6/sentimental-mario-more/mario.py
# Jorge Daniel Gómez Quintana "Sentimental Mario More"

while True:
    try:
        height = int(input("What is the height of the pyramid?"))
        if 1 <= height <= 8:
            break
        else:
            print("Repeat: Height must be between 1 and 8, inclusive.")
    except ValueError:
        print("Please enter a valid integer between 1 and 8.")

for i in range(1, height + 1):
    # Print leading spaces
    print(" " * (height - i), end="")

    # Print left pyramid
    print("#" * i, end="")

    # Print gap
    print("  ", end="")

    # Print right pyramid
    print("#" * i)
